src/main/webapp/app/exercises/programming/manage/update/wizard-mode/programming-exercise-update-wizard.component.scss

```diff
@@ -0,0 +1,70 @@
+.form-step + .form-step:before {
+    content: '';
+    display: block;
+    height: 1px;
+    background: gray;
+    margin-bottom: 1rem;
+}
+
+.code-hint-generation-wrapper {
+    .planned,
+    .finished,
+    .check {
+        color: var(--success);
+    }
+
+    .unset {
+        color: var(--secondary);
+    }
+
+    .current {
+        color: var(--warning);
+    }
+
+    li {
+        display: list-item !important;
+    }
+
+    .connector {
+        width: 100%;
+        min-width: 50px;
+        border-top: 2px solid;
+        margin-top: 1em;
+    }
+}
+
+.status-step {
+    width: 25px;
+    overflow-x: visible;
+    justify-content: flex-start;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .status-step-content {
+        min-width: fit-content;
+        white-space: nowrap;
+
+        .selected-label {
+            font-weight: bold;
+        }
+    }
+
+    .header-icon {
+        z-index: 2;
+        font-size: 2em;
+
+        & > fa-icon {
+            cursor: pointer;
+            background-color: var(--programming-exercise-instruction-step-wizard-card-header-background);
+            width: 30px;
+            height: 30px;
+            text-align: center;
+            padding: 6px 0;
+            font-size: 12px;
+            line-height: 1.428571429;
```

Comment:
why did we chose this number?

Reply:
I have no idea no clue 😅 I just took over the design for the stepper

Reply:
I think if that isn't necessary we should just remove it since in my experience setting `line-height` more often leads to unintended behaviours instead of being useful.

Reply:
I removed it and it still looks the same, so we should be fine with removing it